wurst/_wurst/Assets.wurst

```diff
@@ -16222,3 +16222,380 @@ public class Icons
 	static constant pASBTNVampiricAura = "ReplaceableTextures\\PassiveButtons\\PASBTNVampiricAura.blp"
 	static constant pASBTNVorpalBlades = "ReplaceableTextures\\PassiveButtons\\PASBTNVorpalBlades.blp"
 	static constant pASBTNWarStomp = "ReplaceableTextures\\PassiveButtons\\PASBTNWarStomp.blp"
+
+
+/*
+	Source: cj_order.j v 0.12
+		( Orders Define Library
+		Intended to optimize map script by replacing conversion of strings into order ids. )
+
+    This file was part of cJass standard library
+	2009 Shadow Daemon \\ cjass.xgm.ru
+*/
+
+/** Class that contains every known order in game. */
+public class Orders
+	static constant smart					= 851971
+	static constant stop					= 851972
+	static constant setrally				= 851980
+	static constant getitem					= 851981
+	static constant attack					= 851983
```

Comment:
please fix the mixed indentations here

Reply:
it's strange because in my IDE its all right with indentations.